.github/workflows/conformance-gke-v1.12.yaml

```diff
@@ -118,7 +118,7 @@ jobs:
       github.event_name == 'schedule' ||
       github.event_name == 'pull_request'
     runs-on: ubuntu-latest
-    timeout-minutes: 60
+    timeout-minutes: 75
```

Comment:
Is this bump needed given the below workaround?

Reply:
yeah even with the workaround this run took 1h 1m 36s 🙀  https://github.com/cilium/cilium/actions/runs/3200390404/jobs/5227238248